app/services/replacement_parameters_service.rb

```diff
@@ -30,10 +30,10 @@ def process_replacements_hash(replacements_hash)
 
   def replacements
     {
-        "Client.PreferredName": client&.preferred_name&.titleize,
+        "Client.PreferredName": @intake&.preferred_name&.titleize,
         "Preparer.FirstName": preparer_first_name,
         "Documents.List": documents_list,
-        "Client.LoginLink": intake.is_ctc? ? new_ctc_portal_client_login_url(locale: @locale, host: Rails.configuration.ctc_url) : new_portal_client_login_url(locale: @locale),
+        "Client.LoginLink": @intake.is_ctc? ? new_ctc_portal_client_login_url(locale: @locale, host: Rails.configuration.ctc_url) : new_portal_client_login_url(locale: @locale),
```

Comment:
Consider removing the `@` from `@intake` since the `attr_accessor` lets you. Not a blocker.